src/spec-node/utils.ts

```diff
@@ -116,6 +116,7 @@ export interface DockerResolverParameters {
 	experimentalFrozenLockfile?: boolean;
 	buildxPlatform: string | undefined;
 	buildxPush: boolean;
+	buildxLabel: string | undefined;
```

Comment:
Labels don't depend on `buildx` being available. I suggest to make this similar to `additionalMounts: Mount[];` above, e.g.: `additionalLabels: string[];`.

Reply:
Done